### ISSUE TITLE
[ios] Could not recognize image data when object is captured in portrait | landscape mode. 

### ISSUE THREAD

**Issue by k-saparia** [agoldis/react-native-mlkit-ocr#40]
Hello

Our project is using this library to recognize text from an image. Everything is works as expected on android side but on iOS If I capture image in portrait mode, It does not recognize text. However, here is the catch! If I rotate an object (for instance card) or it has text to 270 degree and then I capture image or text and It will be able to recognize text from images accurately. 

Following this issue: the possible solution would be to able to detect image in normal view (like portrait mode and orientation set to 6)

does anyone facing same issue on iOS? I can not use expo-image-manipulator before processing to OCR!!

**Comment by michaelpol90** [comment]
Yes, IOS have diffrent orientation when you take a picture, check this solution:
https://github.com/agoldis/react-native-mlkit-ocr/issues/13

I don't tested but meybe it will be  work :)

**Comment by rnarang-goauto** [comment]
Couldn't make it work using rotations ?
Was anyone able to solve this ?
